client/config/wdio.config.js

```diff
@@ -140,7 +140,7 @@ const config = {
   mochaOpts: {
     ui: "bdd",
     compilers: ["js:@babel/register"],
-    timeout: 90000
+    timeout: 180000
```

Comment:
Is this change necessary or was it just there for the tests?

Reply:
It is necessary as some tests, especially `"Should create second draft report with conflicts"`, sometimes take more than 120 seconds. I think 180 seconds will keep us on the safe side.